spec/support/shared_contexts/sample_registrations.rb

```diff
@@ -19,6 +18,6 @@
                    business_type: business_type,
                    tier: tier,
                    key_people: key_people,
-                   metaData: { route: route })
+                   metaData: { route: nil })
```

Comment:
Why was it necessary to change this to nil? All registrations should have a route.

Reply:
This is because [this test](https://github.com/DEFRA/waste-carriers-engine/blob/760fc9dbad1448ca1396f174e75048801282d9d3/spec/presenters/waste_carriers_engine/certificate_presenter_spec.rb#L135) is trying to test the metadata route being set, but that context is setting it in an implicit way that stops the test working